uber/templates/registration/merch.html

```diff
@@ -91,8 +91,8 @@
     }
     
     $(function(){
-        $('#message').ajaxError(function(){
-            showTop('Oh noes - the web server is down or something!!!!!');
+        $('#message').ajaxError(function () {
+            toastr.error('Oh noes - the web server is down or something!!!!!');
```

Comment:
This was the only other place where we were using ``showTop`` instead of ``toastr``.

I should probably write a better error message, but that's out of scope for this PR.